README.md

```diff
@@ -54,9 +54,9 @@ For more details on the yotta/mbed OS edition of mbed TLS, including example pro
 
 ### Make
 
-We intentionally only use the minimum of `Make` functionality, as a lot of `Make` features are not supported on all different implementations of Make or on different platforms. As such, the Makefiles sometimes require some manual changes or export statements in order to work for your platform.
+We require GNU Make. To build the library and the sample programs, GNU Make and a C compiler are sufficient. Some of the more advanced build targets require some Unix/Linux tools.
```

Comment:
Our makefile only works with GNU make and I think we're doing developers a disservice not to state it explicitly. “We intentionally only use the minimum of Make functionality” is wrong. We use features that only exist in GNU make. This is why I disagree with the paragraph you want to put back in.